### DISCUSSION TITLE
Suggestion - Weather

### DISCUSSION THREAD

**Discussion by choppygraph** [HelixNebulaStudio/RiseOfTheDead#521] [Ideas]
RoTD always has clear skies.
Weather might be an interesting addition, like rain or thunder.

**Comment by Leroy8888** [comment]
Would it be an idea to add environmental changes to temperature with these changes or is the main goal purely meant to be aesthetic?

**Comment by MXKhronos** [comment]
> Would it be an idea to add environmental changes to tempature with these changes or is the main goal purely meant to be aesthetic?

Aesthetic for now, but for extreme weathers, there's plans for disadvantages, e.g. `heavyrain` could reduce fire DoT effects duration by half or something.
